components/popconfirm/__tests__/__snapshots__/demo.test.js.snap

```diff
@@ -29,6 +29,14 @@ exports[`renders ./components/popconfirm/demo/dynamic-trigger.md correctly 1`] =
 </div>
 `;
 
+exports[`renders ./components/popconfirm/demo/icon.md correctly 1`] = `
+<a
+  href="#"
+>
+  Delete
+</a>
```

Comment:
The popup is not snapshotted.

Reply:
I add an icon demo, when I run with `npm test -- -u`, the snapshot update by itself [icon.md](https://github.com/ant-design/ant-design/pull/11127/files#diff-427e655e7bac176bb0c2d0e05389593a)

Reply:
👌